subprojects/docs/src/docs/release/notes.md

```diff
@@ -18,6 +18,21 @@ The new metadata format is still under active development, but it can already be
 ### Example new and noteworthy
 -->
 
+### Specifying metadata sources for repositories
+
+Gradle now allows you to explicitly state for [which metadata files](userguide/repository_types.html#sub:supported_metadata_sources) it should search in a repository. You can use this, for example, to gain efficiency when querying a Maven repository by telling Gradle not to look for plain `jar` files anymore when the `pom` file for a module is missing.
```

Comment:
> You can use this, for example, to gain efficiency when querying a Maven repository by telling Gradle not to look for plain `jar` files anymore when the `pom` file for a module is missing.

If I understand this correctly, something better might be:

"Use the following to configure Gradle to fail-fast resolving a dependency if a POM file is not found first."

// snippet

"This avoids a 2nd request for the JAR file when the POM is missing, making dependency resolution from Maven repositories faster in this case."

Reply:
Thanks for the suggestion. I changed it.